src/styles.scss

```diff
@@ -1 +1,35 @@
-/* You can add global styles to this file, and also import other style files */
+.flex {
+  display: flex;
+}
+
+html, body { height: 100%; }
+body { margin: 0; font-family: Roboto, "Helvetica Neue", sans-serif; }
+table { width: 100%;}
+tr.data-row:hover { 
+  background-color: whitesmoke;
+}
+
+.list-component {
+  width: 100%; 
```

Comment:
These classes can be moved to where they are going to be used since they are specific